packages/esm-ward-app/src/ward-view-header/ward-metrics.component.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styles from './ward-metrics.scss';
+import { useBeds } from '../hooks/useBeds';
+import { useParams } from 'react-router-dom';
+import { showNotification } from '@openmrs/esm-framework';
+import { useTranslation } from 'react-i18next';
+import { getBedMetrics } from '../ward-view/ward-view.resource';
+import WardMetric from './ward-metric.component';
+
+const bedMetrics = [
+  { name: 'Patients', key: 'patients' },
+  { name: 'Free beds', key: 'freeBeds' },
+  { name: 'Capacity', key: 'capacity' },
+];
+const WardMetrics = () => {
+  const { locationUuid: locationUuidFromUrl } = useParams();
+  const { beds, isLoading, error } = useBeds({ locationUuid: locationUuidFromUrl });
+  const { t } = useTranslation();
+  if (error) {
```

Comment:
This will depend on how the useBeds() hook works, but per the ticket description, we do not want to fail if the bedmanagment module is not installed, so this needs to be aware of the bed management feature flag.  @chibongho should be able to advise here.

Reply:
Hi @mseaton , if bedmanagement module,is not installed,this component is not rendered and there will be a empty page.Is this the desired behaviour or should the metrics be displayed having "--" as value.

Reply:
Thanks @mseaton . Will look into it.